src/ring_theory/localization.lean

```diff
@@ -65,7 +65,10 @@ open function
 set_option old_structure_cmd true
 
 /-- The type of ring homomorphisms satisfying the characteristic predicate: if `f : R →+* S`
-    satisfies this predicate, then `S` is isomorphic to the localization of `R` at `M`. -/
+    satisfies this predicate, then `S` is isomorphic to the localization of `R` at `M`.
+    We later define an instance coercing a localization map `f` to its codomain `S` so
+    that the `R`-algebra instance on `S` can 'know' the map needed to induce the `R`-algebra
+    structure. -/
```

Comment:
```suggestion
satisfies this predicate, then `S` is isomorphic to the localization of `R` at `M`.
We later define an instance coercing a localization map `f` to its codomain `S` so
that the `R`-algebra instance on `S` can 'know' the map needed to induce the `R`-algebra
structure. -/
```
Sometimes VS Code interprets 4-space indentations in markdown docstrings as code blocks.

Reply:
thank you!